dhis-2/dhis-web-api/src/main/resources/org/hisp/dhis/webapi/servlet/login.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE HTML>
+<html class="loginPage" dir="ltr">
+<head>
+  <title>DHIS 2 Demo</title>
```

Comment:
just a small thing, I've never seen a space between `DHIS` and `2`. Not sure if that is correct. Should it be `DHIS2`?